frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1302,7 +1326,7 @@ function ReaderFooter:addToMainMenu(menu_items)
             },
             {
                 text_func = function()
-                    return T(_("Minimum progress bar width: %1".. "\u{202F}%"), self.settings.progress_bar_min_width_pct)
+                    return T(_("Minimum progress bar width: %1\u{202F}%"), self.settings.progress_bar_min_width_pct)
```

Comment:
I thought `\u{XXXX}` doesn't work too well in translated strings ? @Frenzie ?
(Some others below.)

Reply:
I proposed to exclude it from the string but @Frenzie recommended to pass it to translators.
And it is already in the translated string, before this PR.

Reply:
https://github.com/koreader/koreader/pull/11885#issuecomment-2132071079

Reply:
Got it, you mean to include the symbol itself, not the \u code.

Reply:
I didn't specify but I meant as ` `, not as `\u{etc}`.

> I thought \u{XXXX} doesn't work too well in translated strings ? @Frenzie ?

I'm not sure if that's still true; something may have been fixed in Weblate. (Unless it was a text extraction issue; I forget.)

Reply:
Found #10845 - follow the links in these for the previous issues.

Reply:
@mergen3107 , can you please replace `\u{202F}` with `\xE2\x80\xAF`.

Reply:
And add a comment near it saying what it is `-- U+202F NARROW NO-BREAK SPACE` so we don't have to lookup that utf8.